eo-runtime/src/test/eo/org/eolang/rust-tests.eo

```diff
@@ -358,17 +356,14 @@
     """
     use eo_env::EOEnv;
     use eo_env::eo_enum::EO;
-    use eo_env::eo_enum::EO::{EOVertex, EOError};
+    use eo_env::eo_enum::EO::{EOVertex};
 
-    pub fn foo(env: &mut EOEnv) -> EO {
-      let eobook = env.find("$.^.book");
-      let copy = env.copy(eobook as u32).unwrap();
-      let eoline = env.find("$.^.line") as u32;
-      if env.bind(copy.clone(), eoline, "content").is_none() {
-        EOError("bind failed".to_string())
-      } else {
-        EOVertex(copy)
-      }
+    pub fn foo(env: &mut EOEnv) -> Option<EO> {
+      let eobook = env.find("$.^.book")?;
+      let copy = env.copy(eobook)?;
+      let eoline = env.find("$.^.line")?;
+      env.bind(copy.clone(), eoline, "content")?;
+      Some(EOVertex(copy))
```

Comment:
@levBagryansky Should there also be 4 spaces missing at the beginning of the line with the code?